portal/models/i18n_utils.py

```diff
@@ -0,0 +1,163 @@
+"""Module for i18n methods and functionality"""
+from future import standard_library  # isort:skip
+
+standard_library.install_aliases()  # noqa: E402
+
+
+from collections import defaultdict
+import io
+import sys
+import os
+from zipfile import ZipFile
+
+from flask import current_app
+import requests
+from polib import pofile
+
+POT_FILES = (
+    'portal/translations/messages.pot',
+    'portal/translations/js/src/frontend.pot',
+)
+
+
+class BearerAuth(requests.auth.AuthBase):
+    """Add bearer token to HTTP headers for authenticated requests to Smartling"""
```

Comment:
perhaps this is the only location needed, but this utility seems of greater scope than i18n?

Reply:
@pbugni 
Agreed. Any ideas where a better place might be? It's specific to `requests` module unfortunately

Reply:
@ivan-c  how about `requests_utils.py` ?  or just leave it, and we'll move it out when a second need arises

Reply:
I thought we might be able to use it in our tests, but it looks like the test [HTTP client isn't `requests` under the covers](http://werkzeug.pocoo.org/docs/0.14/test/#testing-api), so I'll leave it as-is until we can find another use.